caffe2/python/serialized_test/serialized_test_util.py

```diff
@@ -149,7 +149,7 @@ def parse_proto(x):
         inout_path = os.path.join(temp_dir, 'inout.npz')
 
         # load serialized input and output
-        loaded = np.load(inout_path, encoding='bytes')
+        loaded = np.load(inout_path, encoding='bytes', allow_pickle=True)
```

Comment:
Newest numpy changed the default value to False for security reason, in our case it's pretty safe since the serialized test files were generated by us.